src/Pages/Question/first.jsx

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable consistent-return */
+/* eslint-disable no-return-assign */
+/* eslint-disable no-unused-vars */
+/* eslint-disable react/prop-types */
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import findQuestions from '../../api';
+import QuestionCss from './Question.module.css';
+
+export default function Question() {
+  const [data, setData] = useState([]);
+  const [error, setError] = useState('');
+  const [answer, setAnswer] = useState();
+  const [show, setShow] = useState(false);
+  const [correctAnswer, setCorrectAnswer] = useState(0);
+  const [wrongAnswer, setWrongAnswer] = useState(0);
+  const [disable, setDisable] = useState(false);
+
+  useEffect(() => {
```

Comment:
You don't need to fetch the questions everything the page loads. 
It needs to be fetched once when the page loads. Also I'm seeing you're not using the context API to manage the questions, You should use it.